exercises/sum-of-multiples/example.py

```diff
@@ -1,4 +1,4 @@
 def sum_of_multiples(limit, multiples):
     return sum(value for value in range(limit)
                if any(value % multiple == 0
-                      for multiple in multiples))
+                      for multiple in multiples if multiple))
```

Comment:
While this change technically fixes issue #1567, it would be preferable to also add the new test cases that would cause the old revision to fail. Can you please update `sum_of_multiples_test.py` to the latest canonical data? #1568 is stale, and could be closed if the issue was resolved elsewhere.